src/MUIDataTable.js

```diff
@@ -1349,7 +1362,7 @@ class MUIDataTable extends React.Component {
     } = this.state;
 
     const rowCount = this.state.count || displayData.length;
-    const rowsPerPage = this.options.pagination ? this.state.rowsPerPage : displayData.length;
+    const rowsPerPage = this.state.pagination ? this.state.rowsPerPage : displayData.length;
```

Comment:
Is there a bug with the pagination?

Reply:
Not a bug, but I`m hiding the pagination on printing too